pages/juang961225/style.css

```diff
@@ -0,0 +1,115 @@
+body {
+    display: flex;
+    flex-wrap: nowrap;
+    margin: 0px;
+    padding: 0px;
+}
+.main {
+    align-items: flex-start;
+    background-image: url(../juang961225/pictures/FOR\ THE\ HORDA.jpg);
```

Comment:
hey, buddy, I think you should change the name of the image avoiding to have empty spaces between the words and also they shouldn't have capitalized letters

Reply:
solved